api/aperture/policy/language/v1/fluxmeter.proto

```diff
@@ -12,16 +12,29 @@ import "aperture/policy/language/v1/selector.proto";
 // See also [Flux Meter overview](/concepts/integrations/flow-control/flux-meter.md).
 //
 // :::
-//
-// Example of a selector that creates a histogram metric for all HTTP requests
-// to particular service:
+// Example:
 // ```yaml
-// selector:
+// static_buckets:
+//    buckets: [5.0,10.0,25.0,50.0,100.0,250.0,500.0,1000.0,2500.0,5000.0,10000.0]
+// flow_selector:
 //   service_selector:
-//     service: myservice.mynamespace.svc.cluster.local
-//   flow_selector:
-//     control_point: ingress
-// ```
+//      agent_group: demoapp
+//      service: service1-demo-app.demoapp.svc.cluster.local
+// flow_matcher:
+//   control_point: ingress
+//   label_matcher:
```

Comment:
you can remove label_matcher here - we showed it in the classifier, so it's enough